projects/admin.py

```diff
@@ -70,17 +72,29 @@ def _f(field, name=None):
     no_gender = _f('tn_gender_not_given', _('ohne Angabe'))
     supervisors = Field()
 
-    class Meta:
-        """Define meta properties for the Project export."""
+    def dehydrate_begin_date(self, project: Project):
```

Comment:
```suggestion
    def dehydrate_begin_date(self, project: Project) -> str:
```
We currently have type checks with mypy only enabled for methods, which have a return type defined. So this should be done in cases the input values are type annotated.